.circleci/config.yml

```diff
@@ -0,0 +1,70 @@
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: jgagnaire/scalitysproxydswift
```

Comment:
What's in this image?

Reply:
I've put all the dependencies needed in the image, based on xenial. I've added the Dockerfile content in the description of
https://hub.docker.com/r/jgagnaire/scalitysproxydswift/

Reply:
Then I'd prefer to have the `Dockerfile` in this repository (not in the root, of course, but some suitable dir), and set up a DockerHub 'autobuild' image using it.

Reply:
Done, I have added the Dockerfile to the .circleci/ directory, and configured a public automated build on Docker hub